tests/1-3-conformance.js

```diff
@@ -31,8 +47,44 @@ describe('Basic Conformance', function() {
         this.test.cell.skipMessage = 'Tested by other tests in this suite.';
         this.skip();
       });
-      it('verifiers MUST produce errors when non-conforming documents ' +
-        'are detected.', async function() {
+      it('A conforming document MUST be secured by at least one securing ' +
+        'mechanism as described in Section 4.12 Securing Mechanisms.',
+      async function() {
+        this.test.link = `https://w3c.github.io/vc-data-model/#securing-mechanisms:~:text=A%20conforming%20document%20MUST%20be%20secured%20by%20at%20least%20one%20securing%20mechanism%20as%20described%20in%20Section%204.12%20Securing%20Mechanisms.`;
+        // covers both embedded and enveloped dynamically
+        shouldBeSecured(issuedVc);
+      });
+      it('A conforming issuer implementation MUST include all ' +
+        'required properties in the conforming documents it produces.',
+      async function() {
+        this.test.link = `https://www.w3.org/TR/vc-data-model-2.0/#conformance:~:text=MUST%20include%20all%20required%20properties%20in%20the%20conforming%20documents%20it%20produces`;
+        checkRequiredProperties(issuedVc);
+      });
+      it('A conforming issuer implementation MUST secure the ' +
+        'conforming documents it produces using a securing mechanism' +
+        'described in Section 4.12 Securing Mechanisms.',
+      async function() {
+        this.test.link = `https://www.w3.org/TR/vc-data-model-2.0/#conformance:~:text=MUST%20include%20all%20required%20properties%20in%20the%20conforming%20documents%20it%20produces`;
+        shouldBeSecured(name, issuedVc);
+      });
+      it('A conforming verifier implementation MUST perform ' +
+        'verification on a conforming document as described in' +
+        'Section 4.12 Securing Mechanisms.',
+      async function() {
+        this.test.link = `https://www.w3.org/TR/vc-data-model-2.0/#conformance:~:text=MUST%20include%20all%20required%20properties%20in%20the%20conforming%20documents%20it%20produces`;
+        await assert.doesNotReject(endpoints.verify(issuedVc),
+          'Failed to verify credential.');
+      });
+      it('A conforming verifier implementation MUST check ' +
+        'that each required property satisfies the normative requirements' +
+        'for that property',
+      async function() {
+        this.test.link = `https://www.w3.org/TR/vc-data-model-2.0/#conformance:~:text=MUST%20include%20all%20required%20properties%20in%20the%20conforming%20documents%20it%20produces`;
+        this.test.cell.skipMessage = 'Tested by other tests in this suite.';
```

Comment:
this can be handled by awaiting a previous call, you can do something like this

```js
async function testSomething(){
  endpoints.post('goo');
}

const somethingTest = testSomething();

it('assertion 1', async function() {
  await somethingTest;
})

it('assertion 2', async function() {
  await somethingTest;
})
```

Reply:
I wouldn't go that far. In fact, I realized when going back through the MUSTs just now that all of Section 1 Introduction (of which 1.3 Conformance is a part) is non-normative...so we can remove the `1-3-conformance.js` file altogether (and should for maintainability).

Let's merge this PR first, though.

Reply:
Actually, since you've brought them all in the way you have, we could do what @aljones15 is suggesting. It'll make the code read a little strangely (all the test's will get called outside of any `it()` and then their results awaited in each `it()`), but if y'all feel this will comfort developers and spec reviewers who will see those MUSTs and wonder why they're not tested, I'm more than happy for these to stay.

Just still on the fence as to whether the await approach is worth the effort. Maybe let's discuss it soon.